VTSYoutubeDropper/VTSYoutubeDropper.cs

```diff
@@ -170,7 +170,7 @@ public void WindowFunc(int id)
                 _url = GUILayout.TextField(_url);
                 if (GUILayout.Button($"{_i18nConfirm}"))
                 {
-                    _loader.Start(_url);
+                    _loader.Start(_url, browserType: null, profileFolderName: string.Empty);
```

Comment:
Is it meaning that I should create a dropdown to that users select the browser which they are using?

Reply:
Yes, and yout will need to add a textbox too.

When variable `profileFolderName` is empty, it will use default path on windows system. (Each browser is different.)
If you wants to specific a certain profile folder name or a path of your certain profile folder, just set the value to this variable.

e.g. I have a profile path: `C:\Users\User\Desktop\MyProfile`

```csharp
string profileFolderName = @"C:\Users\User\Desktop\MyProfile";
```

or 

e.g. I have an another profile name: `MyNewProfile`

```csharp
string profileFolderName = "MyNewProfile";
```

Reply:
FYI, due to it will need the SQLite's DLL file to get it to work.

So you better had a check when you made a new release, the program whether or not to find that DLL file.

Reply:
I think I will only add a dropdown to support `Default` folder.

Let user to input the folder name is not a good idea.

The names between browser and folder can't match directly. 

I can image that user will ask me how to setup the profile folder even I write the document. 😂

![image](https://github.com/MurabitoB/VTSYoutubeDropper/assets/22661940/c2bef955-503d-4f6b-9de1-4606033be230)

Reply:
That is true. Even if you already write it in the document, but users may still have questions about it.

But force use `Default` may face the problem, although the default profile folder is named as `Default`, But not everyone's profile folder is `Default` at all.

And other browser like Mozilla Firefox, the default profile structure is really different from Chromium series browser.

These may be the issue.

I would suggest to use a textbox to instead of drop-down could be better.
 But, it's all up to you. Just make it what you like it to be!